docs/source/tutorials/faq.ipynb

```diff
@@ -271,6 +271,18 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "13228a99-5d3f-47c0-87e5-2290d16461c4",
+   "metadata": {},
+   "source": [
+    "Methods that call `filter.find_label_issues` use `label_issues_batched` internally if `low_memory=True` is passed as a parameter. The following methods provide this option - \n",
+    "\n",
+    "1. [classification.CleanLearning](https://docs.cleanlab.ai/stable/cleanlab/classification.html?highlight=cleanlearning#cleanlab.classification.CleanLearning)\n",
+    "2. [multilabel_classification.filter.find_label_issues](https://docs.cleanlab.ai/stable/cleanlab/multilabel_classification/filter.html#cleanlab.multilabel_classification.filter.find_label_issues)\n",
+    "3. [token_classification.filter.find_label_issues](https://docs.cleanlab.ai/stable/cleanlab/token_classification/filter.html?highlight=token#cleanlab.token_classification.filter.find_label_issues)"
```

Comment:
all links should be relative links, see instructions on how to achieve this here:

https://github.com/cleanlab/cleanlab/blob/master/DEVELOPMENT.md#relative-link-formatting-instructions